pkg/controller/daemonset/daemonset_util.go

```diff
@@ -254,7 +254,10 @@ func unavailableCount(ds *appsv1alpha1.DaemonSet, numberToSchedule int) (int, er
 	}
 	r := ds.Spec.UpdateStrategy.RollingUpdate
 	if r == nil {
-		return 0, nil
+		return 1, nil
```

Comment:
This should not be changed.

Reply:
I just refer to the implementation of max surge and return the default value of maxunavilable in the doc.
Sorry to change this because I did not read all codes in ds reconcile function.

Reply:
Why I can not set default value when rollingupdate is nil?
This code is necessary when the webhook is unreachabole in my case.